docs/getting_started/index.rst

```diff
@@ -227,12 +270,19 @@ sub-module), check the style, and make sure the docs render correctly
 3. Push your branch to your fork on github
 ++++++++++++++++++++++++++++++++++++++++++
 
-(sometimes refered to as
-"publishing" since it becomes public, but only in your fork) by running
+The first time you push a new branch, you need to specify to which remote the branch
+should be pushed. Normally this will be ``origin``:
 
-.. code-block:: sh
+.. code-block:: console
+
+   $ git push -u origin implement_feature_1
```

Comment:
could also say to set 
```
git config --global push.autoSetupRemote true
```
so this is done automatically

Reply:
Added a footnote about this